src/SplitText.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export const SplitText = ({ children, className, separator, ...rest }) => {
+    const words = children.split(separator);
+
+    return words.map((word, index) => {
+        const withSeparator = index !== words.length - 1;
```

Comment:
Maybe we should rethink this. Previously we were also checking if the last string char was a space and rendering it accordingly, for example:
![imagem](https://user-images.githubusercontent.com/11398440/80193930-87da0600-8611-11ea-9ada-78cb1afccefb.png)

![imagem](https://user-images.githubusercontent.com/11398440/80194029-b48e1d80-8611-11ea-8ebc-f63e82e93cde.png)

![imagem](https://user-images.githubusercontent.com/11398440/80194048-bb1c9500-8611-11ea-88f7-42468b05020a.png)

I think we should still respect all those spaces, including the last one, if it is one.
Although I think we should check if the current word is an `empty` string, if it is `withSeparator` should be true

Reply:
You are right, I've added it and now is 👍:

![Screenshot 2020-04-24 at 10 47 16](https://user-images.githubusercontent.com/33324750/80199237-fff7fa00-8618-11ea-9dd5-d2b3d5e58464.png)